Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -2029,3 +2033,8 @@ void MainWindow::set_facegraph_mode_adapter(bool is_polyhedron)
   else
     set_face_graph_default_type(SURFACE_MESH);
 }
+
+void MainWindow::resetStatusColor()
+{
+  statusBar()->setStyleSheet("color: black");
+}
```

Comment:
The default color may not be black. If the user uses a "Dark" style, then the default color might be closer to white. Is not their a way to restore the default style sheet?